styles/markov.css

```diff
@@ -0,0 +1,174 @@
+/* General styles used in multiple elements */
```

Comment:
Obviously not ideal to have a single big CSS file in basically random order, but it's a start.